### PR TITLE
updates to device auth

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** updates to device auth
- **Is this PR related to a Monolith release?** No

Preview Link:
https://6647ae5a1ee1d0224700da9f--reverent-murdock-829d24.netlify.app/docs/guides/device-authorization-grant/main/

### Resolves:

* [OKTA-727783](https://oktainc.atlassian.net/browse/OKTA-727783)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/device-authorization-grant/main/index.md](https://github.com/okta/okta-developer-docs/blob/4c4a961bbc7a9aff65a2cbb3694a99fbdc02933d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdevice-authorization-grant%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c0274497-4d39-499e-9dc6-08a1e271eb55) | 73  | 84  | 84  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.c1f37efe-0c07-4467-bcdd-f136f4a3e522?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by susanharper-okta** [comment]
approved by @shantanusardal-okta on Slack.

File: packages/@okta/vuepress-site/docs/guides/device-authorization-grant/main/index.md [hunk 1]
```diff
@@ -18,32 +18,25 @@ This guide describes the Device Authorization feature, which allows users to sig
...
-1. In the left navigation pane of the Admin Console, go to **Applications** > **Applications**.
+1. Open the **Admin Console** for your org.
+1. Go to **Applications** > **Applications** to view the current app integrations.
 1. Click **Create App Integration**.
-1. On the Create a new app integration page, select **OIDC - OpenID Connect** as the **Sign-in method**, and then pick **Native Application**.
-1. On your native application page, fill in the application settings. Ensure that you select **Device Authorization** as the allowed **Grant type** in the General Settings.
+1. Select **OIDC - OpenID Connect**  as the **Sign-in method**, and then **Native Application** as the **Application type**.
+1. Click **Next** and specify the **App integration name**.
+1. Select **Device Authorization** and **Refresh Token** as the grant types.
+1. Select **Allow everyone in your organization to access** in the **Assignments** section, then click **Save**.
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/guides/device-authorization-grant/main/index.md:39]:
```suggestion
1. Select **Allow everyone in your organization to access** in the **Assignments** section, and then click **Save**.
```

**Review by grahamsmith-okta — APPROVED** [review]
one suggestion but otherwise approved

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/device-authorization-grant/main/index.md](https://github.com/okta/okta-developer-docs/blob/055fd955f782555db65217d096f09e314c24cbdc/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdevice-authorization-grant%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/613ab1a4-a2cc-43b6-8e32-95e5091cc11d) | 73  | 84  | 84  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.53af84d5-3f66-400d-8987-63aa3e97c357?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by grahamsmith-okta — APPROVED** [review]

**Pull request merged**